bitnami/elasticsearch/values-production.yaml

```diff
@@ -44,6 +44,8 @@ master:
   replicas: 3
   heapSize: 128m
   antiAffinity: "soft"
+  ## node affinity
```

Comment:
Can you use uppercase in those sentences? In this way we can differentiate the comment from the variable, if not they are pretty similar